From 5268936259c598ea1e51370df33011a48ec9f3d4 Mon Sep 17 00:00:00 2001
From: Gaetan Nadon <memsize@videotron.ca>
Date: Fri, 13 Feb 2015 19:37:16 -0500
Subject: [PATCH 2/8] Remove SCO support for SHELL_CMD and startx man page.

SCO support was removed from startx.cpp and xinitrc.cpp earlier.

    Remove unixware / sco support
    http://cgit.freedesktop.org/xorg/app/xinit/commit/
    ?id=fdf03cd2fdfd9cd5635334c5e4dc2bb23e92e37a

Signed-off-by: Gaetan Nadon <memsize@videotron.ca>
Reviewed-by: Alan Coopersmith <alan.coopersmith@oracle.com>
---
 configure.ac    |  5 -----
 man/Makefile.am |  1 -
 man/startx.man  | 26 --------------------------
 3 files changed, 32 deletions(-)

diff --git a/configure.ac b/configure.ac
index 18794b3..461648b 100644
--- a/configure.ac
+++ b/configure.ac
@@ -190,13 +190,8 @@ case $host_os in
     *solaris*)
 	SHELL_CMD="/bin/ksh"
 	;;
-    *sco*)
-	SHELL_CMD="/bin/ksh"
-	SCOMAN=1
-	;;
 esac
 AC_SUBST(SHELL_CMD)
-AC_SUBST(SCOMAN)
 AC_SUBST(XSERVERNAME)
 AC_SUBST(XCONFIGFILE)
 AC_SUBST(XCONFIGFILEMAN)
diff --git a/man/Makefile.am b/man/Makefile.am
index b24faca..9c6569f 100644
--- a/man/Makefile.am
+++ b/man/Makefile.am
@@ -13,7 +13,6 @@ MAN_SUBSTS+=	-e 's|__XSERVERNAME__|$(XSERVERNAME)|g' \
 		-e 's|__xinitdir__|$(XINITDIR)|g' \
 		-e 's|__bindir__|$(bindir)|g' \
 		-e 's|__libdir__|$(libdir)|g' \
-		-e 's|__SCOMAN__|$(SCOMAN)|g' \
 		-e 's|__configdir__|$(XINITDIR)|g'
 
 
diff --git a/man/startx.man b/man/startx.man
index 0405be0..4728a25 100644
--- a/man/startx.man
+++ b/man/startx.man
@@ -74,7 +74,6 @@ startx -- -dpi 100
 .PP
 startx -- -layout Multihead
 .RE
-.if '__SCOMAN__'' .ig
 .PP
 To determine the client to run,
 .B startx
@@ -90,20 +89,6 @@ looks for the following files, in order:
 .I __xinitdir__/xinitrc
 .RE
 .PP
-..
-.if !'x.__SCOMAN__'x.' .ig
-.PP
-To determine the client to run,
-.B startx
-first looks for a file called
-.I .xinitrc
-in the user's home directory.  If that is not found, it uses
-the file
-.I xinitrc
-in the
-.I xinit
-library directory.
-..
 If command line client options are given, they override this
 behavior and revert to the
 .BR xinit (__appmansuffix__)
@@ -187,17 +172,6 @@ and
 .IR Xsecurity (__miscmansuffix__)
 manual pages for more information on X client/server authentication.
 .SH FILES
-.if '__SCOMAN__'' .ig
-.TP 25
-.I $(HOME)/.startxrc
-Client to run.  Typically a shell script which runs many programs in
-the background.
-.TP 25
-.I __libdir__/sys.startxrc
-Client to use if the user has no
-.I .startxrc
-file.
-..
 .TP 25
 .I $(HOME)/.xinitrc
 Client to run.  Typically a shell script which runs many programs in
-- 
1.9.0

